src/PowerSystem/PowerSystem.jl

```diff
@@ -0,0 +1,70 @@
+module PowerSystem
+
+using Printf
+using SparseArrays
+using ..ExaPF: ParsePSSE, ParseMAT, IndexSet, Spmat
+
+import Base: show
+
+const PQ_BUS_TYPE = 1
+const PV_BUS_TYPE = 2
+const REF_BUS_TYPE  = 3
+
+abstract type AbstractNetworkAttribute end
```

Comment:
At some point, we should discuss the attributes we need here, for the power system.